extra.cabal

```diff
@@ -36,6 +36,7 @@ library
         filepath,
         process,
         clock >= 0.7,
+        semigroups,
```

Comment:
This is for GHC 7.10 since `NonEmpty` wasn't added to `base` then.

Reply:
Can we get a comment to that effect, so I know when I upgrade to GHC 8 only I can delete this part?